.golangci.yml

```diff
@@ -0,0 +1,25 @@
+run:
+  deadline: 10m
+
+linters:
+  disable-all: true
+  enable:
+    - asciicheck
+    - bidichk
+    - durationcheck
+    - exportloopref
+    - gofmt
+    - goimports
+    - gosimple
+    - govet
+    - grouper
+    - ineffassign
+    - makezero
+    - nosprintfhostport
+    - reassign
+    - rowserrcheck
+    - sqlclosecheck
+    - tparallel
+    - typecheck
+    - unconvert
+    - vetshadow
```

Comment:
From what I can tell, **vetshadow** is enabled by default with **govet**. They don't even document it as a separate linter. Can remove?

---

#### For documentation, the new linters added in the pull request are...
bidichk: Checks for dangerous unicode character sequences
durationcheck: check for two durations multiplied together
exportloopref: checks for pointers to enclosing loop variables
grouper: An analyzer to analyze expression groups, e.g. const, var
nosprintfhostport: Checks for misuse of Sprintf to construct a host with port in a URL
reassign: Checks that package variables are not reassigned in another package
tparallel: detects inappropriate usage of t.Parallel() method in your Go test codes
typecheck: Like the front-end of a Go compiler, parses and type-checks Go code